README.md

```diff
@@ -143,28 +166,18 @@ You can then run the integration tests against this stack with `pytest tests/ --
 
 Alternatively you can go the fast way and just run the whole stack + integration tests in docker: 
 
-```
-make integration--test
+```bash
+make integration-test
 ```
 
 #### Unit Tests
 
 If you just want the unit tests simply run:
  
-```
+```bash
 make test
 ```
 
-
-### Pre Commit Hooks
-
-To install pre commit hooks run:
-
-```
-pip install pre-commit
-pre-commit install
-pre-commit install-hooks
-```
 ## Additional information
 
 You can find additional information, development cycle and release information on our [wiki page](https://github.com/real-digital/esque/wiki).
```

Comment:
Do we need to update the wiki as well?

Reply:
Actually I was about to suggest to drop the wiki. Currently it holds no information which isn't in the readme. What do you think @ognjen-j ?